webapp/channels/src/i18n/en.json

```diff
@@ -4167,7 +4167,7 @@
   "more_channels.join": "Join",
   "more_channels.joining": "Joining...",
   "more_channels.next": "Next",
-  "more_channels.noMore": "No results for \"{text}\"",
+  "more_channels.noMore": "No more channels to join",
```

Comment:
Have we changed the Default message?

Reply:
that is the weird part, the default message remained the same https://github.com/mattermost/mattermost-server/blob/master/webapp/channels/src/components/searchable_channel_list.jsx#L170 after the revert that was done https://github.com/mattermost/mattermost-webapp/commit/4437e9dfb4de815e50061670aabffbc9e396b9d6#diff-1a490f08504158838befaa816ee8d5c08dff9e694d5a568ce4b0a4398dcc563eR4163 . Probably a conflict during the monorepo migration, this text changed

Reply:
I think the problem there might be that the check-i18n CI job only checks that the string exists in both en.json and the web app code, not that the defaultMessage value matches the en.json. Similarly, Weblate only notifies translators of changes when en.json changes